projects/aca-content/src/lib/components/search/search-results-row/search-results-row.component.ts

```diff
@@ -114,4 +151,10 @@ export class SearchResultsRowComponent implements OnInit, OnDestroy {
     event.stopPropagation();
     this.store.dispatch(new NavigateToFolder(this.node));
   }
+
+  private stripHighlighting(highlightedContent: string): string {
+    return highlightedContent && highlightedContent !== ''
+      ? highlightedContent.replace(new RegExp(this.highlightPrefix, 'g'), '').replace(new RegExp(this.highlightPostfix, 'g'), '')
```

Comment:
I'm wondering - instead of two calls of replace function and two regexps can we use OR inside regexp so then it will match both - prefix and postfix at once? Is that possible?

Reply:
I tired but for this to work we would need to escape every special character in the prefix and postfix, and the plan is to start taking them from config file which won't be that easy to implement, we would have to write another function to escape each special character and perform another replace

Reply:
ok, then you can leave as it is, thanks for explanation